{{cookiecutter.project_name}}/src/{{cookiecutter.__project_slug}}/__init__.py

```diff
@@ -16,7 +20,9 @@
 {%- else %}
 __all__ = [
 {%- endif %}
+{%- if cookiecutter.vcs %}
     "__version__",
+{%- endif %}
```

Comment:
Should always be there
```suggestion
    "__version__",
```